indico/htdocs/sass/modules/_event_display.scss

```diff
@@ -20,6 +20,8 @@
 @import 'partials/buttons';
 
 
+$header-blue = #1a64a0;
```

Comment:
Did you purposefully put this outside of `_palette.scss` to prevent it from being used in any other place?